README.md

```diff
@@ -7,6 +7,16 @@ Common code shared among
 [MySQL](https://github.com/juttle/juttle-mysql-adapter/)
 and [SQLite](https://github.com/juttle/juttle-sqlite-adapter/) adapters.
 
+#### List of optimized operations
+
+* any filter expression `read sql` (note: `read sql | filter ...` is not optimized)
```

Comment:
Thanks! Let's also reflect what's supported vs not in the README, and we are good.